pkg/chrootuser/user.go

```diff
@@ -1,4 +1,4 @@
-package main
+package chrootuser
```

Comment:
Is this the version Nalin developed for buildah? I'm not sure it is

Reply:
It probably came from their originally.  I should grab the latest from buildah and then we should make buildah vendor in this code.